.github/workflows/build.yml

```diff
@@ -6,17 +6,39 @@ on:
 jobs:
   build-chainlink:
     runs-on: ubuntu-20.04
+    if: ${{ ! startsWith(github.head_ref, 'release/') && ! startsWith(github.ref_name, 'chore/') }}
     steps:
       - name: Checkout repository
         uses: actions/checkout@9bb56186c3b09b4f86b1c65136769dd318469633 # v4.1.2
 
+      - uses: dorny/paths-filter@de90cc6fb38fc0963ad72b210f1f284cd68cea36 # v3.0.2
+        id: change
+        with:
+          predicate-quantifier: every
+          filters: |
+            changelog-only:
+              - 'CHANGELOG.md'
+              - '!common/**'
+              - '!contracts/**'
+              - '!core/**'
+              - '!crib/**'
+              - '!dashboard-lib/**'
+              - '!fuzz/**'
+              - '!integration-tests/**'
+              - '!internal/**'
+              - '!operator_ui/**'
+              - '!plugins/**'
+              - '!tools/**'
+
       - name: Build chainlink image
+        if: ${{ steps.change.outputs.changelog-only == 'true' }}
```

Comment:
If changlog-only is true, then we _don't_ want to build, right? Also do we have a required check that won't be satisfied if this step doesn't run?

Reply:
ahh yeah good catch!